### ISSUE TITLE
Flash of unstyled content

### ISSUE THREAD

**Issue by makshh** [squidfunk/mkdocs-material#558]
## Description

Flash of unstyled content (especially in Firefox)
Internet: 3G

### Expected behavior

No flash of unstyled content

### Actual behavior

Flash of unstyled content

### Steps to reproduce the bug

1. Hard refresh on Firefox, Chrome is less obvious (I didn't test it on high internet speed)

### Package versions

* MkDocs: 0.17.1
* Material: 2.0.1

### Project configuration

Defaults, GIF from GitHub page.

### System information

* OS: OSX
* Browser: Firefox, Chrome (no extensions)

![mkdocs-material-fouc](https://user-images.githubusercontent.com/5025950/32283874-e0f226f6-bf25-11e7-8dc7-6a5ab4e95cae.gif)

I was able to fix this, but I'm not sure if it's only that.

Fix: move loading `modernizr` after application `css` `links`

```html
<link rel="stylesheet" href="../../assets/stylesheets/application-462a8722ef.css">
<link rel="stylesheet" href="../../assets/stylesheets/application-23f75ab9c7.palette.css">
<script src="../../assets/javascripts/modernizr-e826f8942a.js"></script>
```
### [StackOverflow](https://stackoverflow.com/a/6272728/3258604)

**Comment by squidfunk** [comment]
Awesome, thanks!

**Comment by squidfunk** [comment]
Fixed in #561
